src/styles/globals.css

```
@@ -1,8 +1,12 @@
 @import "tailwindcss";
 
 :root {
-  --background: hsl(0 0% 100%);
+  --background: hsl(240 5% 96%);
   --foreground: hsl(240 10% 3.9%);
+  --link: hsl(230 70% 50%);
+  --link-hover: hsl(230 70% 40%);
+  --font-heading: "Noto Sans JP", system-ui, -apple-system, sans-serif;
+  --font-mono: "JetBrains Mono", ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, monospace;
```

Stylelint (17.3.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)